app/domain/repositories/message.go

```diff
@@ -0,0 +1,8 @@
+package repositories
+
+import "github.com/limechain/hedera-eth-bridge-validator/app/persistence/message"
+
+type MessageRepository interface {
+	Get(txId, signature string) ([]message.TransactionMessage, error)
```

Comment:
Good idea should be to explain what you get in the function name (GetByTxIdAndSignature).